package.json

```diff
@@ -119,6 +119,9 @@
 	},
 	"private": true,
 	"repository": "git@github.com:3YOURMIND/kotti",
+	"resolutions": {
```

Comment:
for the reminder: we added this because `vue-tippy` is relying on `tippy.js@4.x`